index.html

```diff
@@ -132,7 +132,7 @@ <h3>Locales and Internationalization</h3>
         
         <p class="definition"><dfn>Unicode Locale</dfn>. A combination of language tag extensions ([[RFC6067]], [[RFC6497]]) and additional processing rules defined by [[CLDR]] to support <a>locales</a>.</p>
         
-        <p>A Unicode locale provides the ability to specify in a language tag international preference variations that go beyond linguistic or regional variation or to select formatting behavior or content when there are multiple options.  Unicode locale identifiers are identical to language tags, but apply additional rules about the content of certain language tags. Unicode Locales increasingly form the basis for <a>internationalization</a> on the Web, particularly as part of the <code>Intl</code> locale framework [[ECMA-402]] in JavaScript [[ECMASCRIPT]].</p>
+        <p>A Unicode locale provides the ability to specify in a language tag international preference variations that go beyond linguistic or regional variation or to select formatting behavior or content when there are multiple options.  Unicode locale identifiers are language tags, but may apply additional rules about the structure and interpretation of certain language tags. Unicode Locales increasingly form the basis for <a>internationalization</a> on the Web, particularly as part of the <code>Intl</code> locale framework [[ECMA-402]] in JavaScript [[ECMASCRIPT]].</p>
```

Comment:
it's not "may apply additional rules": they *do* apply. I like the addition of "interpretation" though.

Reply:
Does this
https://github.com/w3c/ltli/pull/19/commits/118b7aa92788fe6a4e5e1cab5af417d536dce1b3
work for you?

Reply:
It does, although I'm increasingly minded to split the sentence into two parts:

> Unicode locale identifiers are well-formed BCP47 language tags.

(this emphasizes the "language tag-ness" of them)

And then:

> Unicode locale identifiers have additional rules about the structure and content of a language tag as well as supplying specific interpretation of certain subtags.

Reply:
Done.